### PR TITLE
feat: random stuff

### LINKED ISSUE #137: random shit

### PR DESCRIPTION
**Pull request by retinfai**
## Description

Resolves #137 

This PR does the following:

- [x] Add Images to all activities
- [x] Add initial and goal graphs to all

### CODE REVIEW

File: src/data/dummyData.ts [hunk 1]
```diff
@@ -116,7 +108,7 @@ export const DUMMY_DATA_QUESTIONS: Question[] = [
         "All changes have already been added and staged, now we need to commit and push our changes onto main.",
         "Use git commit commands in the terminal to complete this task and click submit to check!"
       ],
-      desiredGraphImgUrl: "ActivityCommitGraph.png"
```
Comment (Bac0nEater) [inline src/data/dummyData.ts:119]: did you test if it works? Cause idk do you need to specify the path.

**Review by Bac0nEater — APPROVED** [review]

File: src/data/dummyData.ts (hunk 1 above)
Comment (retinfai) [reply src/data/dummyData.ts:119]: It isn't implemented yet, and that was already like that, so I modelled all the other URLs like that

Comment (Bac0nEater) [reply src/data/dummyData.ts:119]: You don't need to specify the path, if you put the images in the Public folder, instead of assert

**Review by AoMedium — APPROVED** [review]
Images look good!

File: src/data/dummyData.ts (hunk 1 above)
Comment (AoMedium) [reply src/data/dummyData.ts:119]: yeah currently its in the assets folder (see #139), but i can move it to public if u want, its a very minor thing tho

**Pull request merged**